helm/reana/README.md

```diff
@@ -48,6 +48,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `ingress.annotations.kubernetes.io/ingress.class`        | Type of ingress controller                                                           | traefik                                         |
 | `ingress.annotations.traefik.frontend.entryPoints`       | Entrypoints allowed by the ingress controller                                        | "http,https"                                    |
 | `ingress.enabled`                                        | Create an ingress resource to access the REANA instance from outside the cluster     | true                                            |
+| `kubernetes_jobs_memory_limit`                           | Maximum memory user's Kubernetes job containers can consume. Exceeding the limit will cause the container's termination. For possible value formats see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | None |
```

Comment:
Maximum memory limit for user job containers. Exceeding this limit will terminate the container. Please see the following URL for possible values:  https://foo

Reply:
... also, please add a line to `CHANGES.rst` about this